src/collective/fingerpointing/upgrades/v2/__init__.py

```diff
@@ -1,19 +1,19 @@
 # -*- coding: utf-8 -*-
 from collective.fingerpointing.config import PROJECTNAME
-from collective.fingerpointing.logger import logger
+from collective.fingerpointing.logger import log_info
 
 
 def update_configlet(setup_tool):
     """Update control panel configlet."""
     profile = 'profile-{0}:default'.format(PROJECTNAME)
     setup_tool.runImportStepFromProfile(profile, 'controlpanel')
-    logger.info('Control panel configlet updated.')
+    log_info.logger.info('Control panel configlet updated.')
 
 
 def update_user_actions(setup_tool):
     """Update user actions."""
     profile = 'profile-{0}:default'.format(PROJECTNAME)
     setup_tool.runImportStepFromProfile(profile, 'rolemap')
-    logger.info('Permissions updated.')
+    log_info.logger.info('Permissions updated.')
     setup_tool.runImportStepFromProfile(profile, 'actions')
-    logger.info('User actions updated.')
+    log_info.logger.info('User actions updated.')
```

Comment:
we don't want this information logged into the audit log, neither.

Reply:
you can see this message being logged in the first screenshot (the one for Plone 4).

Reply:
see above